docs/console/auth/import.md

```diff
@@ -0,0 +1,57 @@
+
+This feature is particularly useful if you’re trying to do the following:
+
+- Enable Amplify categories (such as API, Storage, and function) for your existing user base
+- Incrementally adopt Amplify for your application stack
+- Independently manage Amazon Cognito resources while working with Amplify
+
+## Import an existing Cognito user pool
+To successfully import your user pool, your user pools require at least two app clients with the following conditions:
+
+- At least one “Web app client”: an app client without a client secret
+- At least one “Native app client“: an app client with a client secret
+To complete the import procedure, run the `amplify push` command.
+
+[Learn more about getting started with User Pools](https://docs.aws.amazon.com/cognito/latest/developerguide/getting-started-with-cognito-user-pools.html)
+or 
+[configuring a User Pool App Client](https://docs.aws.amazon.com/cognito/latest/developerguide/user-pool-settings-client-apps.html)
+
+## Import an existing identity pool
+In order to successfully import your identity pool, it must have both of the user pool app clients must meet the following requirements:
+
+- An Authenticated Role with a trust relationship to your identity pool
+- An optional Unauthenticated Role if you want to use any guest user access for your Amplify categories (For example, guest access for your Amazon S3 buckets or REST API endpoints).
+
+These roles are usually automatically configured when you create a new identity pool enabling “Unauthenticated” access and have an Amazon Cognito user pool as an authentication provider.
+
+[Learn more about getting started with Identity Pools.](https://docs.aws.amazon.com/cognito/latest/developerguide/getting-started-with-identity-pools.html)
+
+## Import and update your source code
+
+Choose **Import** on the bottom of the page to complete the import procedure. Update your source code by running the following command: 
+```bash
+amplify pull
+```
+Next, generate the necessary GraphQL files by running the following command:
+```bash
+amplify codegen add
+```
+
+After running this command, the following occurs:
+- Your Amplify Library configuration files (aws-exports.js, amplifyconfiguration.json) are automatically populated with your chosen Amazon Cognito resource information
+- Your designated existing Amazon Cognito resource is provided as the authentication and authorization mechanism for all auth-dependent categories (API, Storage and more)
+- Lambda functions are enabled to access the chosen Amazon Cognito resource if you permit it.
+
+## Multi-environment support
+When you create a new environment, Amplify assumes, by default, that you’re managing your app’s Amazon Cognito resources outside of an Amplify project. You’ll be asked to either import a different Amazon Cognito resource or maintain the same Amazon Cognito resource for your app’s auth category.
```

Comment:
Too many commas here: When you create a new environment, you’ll be asked to either import a different Amazon Cognito resource or maintain the same Amazon Cognito resource for your app’s auth category.

Also functionality-wise, what happens when I clone an env in the Amplify console?